doc/rfcs/login_block_sliding_window.md

```diff
@@ -0,0 +1,66 @@
+# Login Sliding-Window Blocks
+
+## Overview and motivation
+`ms-users` service supports some IP and username blocks. The Current version uses block on TTL'ed counter in the Redis database,
+but this brings some strong limits for users. Service monitors count of login attempts from IP address and IP + Username,
+if one of the counters becomes greater than specified config parameter, all requests from this IP or IP+username blocked
+for TTL. For better user experience introducing a new feature
+
+## Sliding window Block
+Solution based on `inredis-ratelimiter`(https://github.com/3hedgehogs/inredis-ratelimiter) with some additions and fixes, replaces the standard block counter(int key) +TTL.
```

Comment:
The solution is based on <...>
<...> standart block counter is replaced with _something_